src/lib/references.ts

```diff
@@ -245,12 +247,22 @@ async function resolveContentRefInSpace(spaceId: string, contentRef: ContentRef)
         baseUrl += '/';
     }
 
-    return resolveContentRef(contentRef, {
+    const resolved = await resolveContentRef(contentRef, {
         space,
         revisionId: space.revision,
         pages,
         baseUrl,
     });
+
+    if (!resolved) {
+        return null;
+    }
+
+    return {
+        ...resolved,
+        text: space.title,
+        subText: resolved.text,
```

Comment:
Why not the reverse? using `text: space.title` works well hen you point to a first page where the title doesn't matter, but it's bit worse when pointin to another page.

```suggestion
        subText: space.title,
```